stable/artifactory-ha/README.md

```diff
@@ -811,6 +811,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.service.type`        | Nginx service type                | `LoadBalancer`                                         |
 | `nginx.service.clusterIP`   | Specific cluster IP or `None` for headless services     | `nil`                         |
 | `nginx.service.loadBalancerSourceRanges`| Nginx service array of IP CIDR ranges to whitelist (only when service type is LoadBalancer) |        |
+| `nginx.service.labels`       | Nginx servicelabels     | `{}`                                                           |
```

Comment:
You are missing a space in servicelabels

Reply:
@danielezer fixed